pkg/metrics/grafana/tidb.json

```diff
@@ -19023,7 +19023,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "histogram_quantile(0.95, sum(rate(tidb_tikvclient_range_task_push_duration_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (le, instance))",
+              "expr": "histogram_quantile(0.95, sum(rate(tidb_tikvclient_range_task_push_duration_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type=~\"resolve-locks.*\"}[1m])) by (le, instance))",
```

Comment:
How about filtering by something like `!~".*pipelined-dml.*"`

Reply:
Done, test it ok

![image](https://github.com/pingcap/tidb/assets/9587680/6a3ca666-2b01-4b2f-911e-b745c4563f83)